runtime/src/accounts_db.rs

```diff
@@ -298,12 +298,18 @@ impl AccountStorageEntry {
             status = AccountStorageStatus::Available;
         }
 
-        if count > 0 {
-            *count_and_status = (count - 1, status);
-        } else {
-            warn!("count value 0 for slot {}", self.slot_id);
```

Comment:
This actually happened locally in the way-before past, locally.

If this ever happens, it's very likely for this to cause the disastrous event of ledger inconsistency by continuing silently like this without being `panic!`-ed.